build-system/server/new-server/transforms/transform.ts

```diff
@@ -26,7 +26,7 @@ const argv = minimist(process.argv.slice(2));
 const transforms = [transformStories(), transformScriptPaths(), transformCss()];
 
 export async function transform(fileLocation: string): Promise<string> {
-  if (argv.esm) {
+  if (argv.esm || argv.sxg) {
     transforms.unshift(transformModules());
```

Comment:
Is this related or just something you noticed as you were doing this PR?

Reply:
I was going to change the script tag to point to a top level directory, but after discussing with @jridgewell this no longer seems necessary.